### PR TITLE
cat value leaderboard

### PR DESCRIPTION
**Pull request by VishramKidPG123**
~~hopefully this isn't broken~~
some of my names may want to be changed
would arrays and for loops for the buttons make this easier? idk

### CODE REVIEW

**Comment by milena-kos** [comment]
not sure if i want this tbh

File: main.py [hunk 1]
```diff
@@ -2541,12 +2565,17 @@ async def lb_handler(interaction, type, do_edit=None):
...
+        await lb_handler(interaction, "val")
+    
     async def slowlb(interaction):
         await lb_handler(interaction, "slow")
 
     async def fastlb(interaction):
         await lb_handler(interaction, "fast")
-
     async def catlb(interaction):
         await lb_handler(interaction, "main")
```
Comment (milena-kos) [inline main.py:2580]: isnt this a duplicate?

File: main.py [hunk 2]
```diff
@@ -2443,10 +2446,24 @@ async def lb_handler(interaction, type, do_edit=None):
...
+                value = get_time(message.guild.id, i, time_type)
+                if int(value) < 0:
+                    set_time(message.guild.id, i, int(default_value), time_type)
+                    continue
             if str(value) != default_value:
                 # round the value (for time dislays)
-                thingy = round((value / devider) * 100) / 100
-
+                thingy = round(value / devider, 2)
+
```
Comment (milena-kos) [inline main.py:2466]: a

**Comment by milenakos** [comment]
this is also conflicting cus of db changes

**Comment by milenakos** [comment]
lb code is gonna get changed quite a bit soon so this will be two breaking changes behind
im gonna close this

**Pull request closed without merging**